### ISSUE TITLE
NMS and EVALUATION report

### ISSUE THREAD

**Issue by maisuria-Anand** [tensorflow/models#10663]
# Prerequisites

Please answer the following question for yourself before submitting an issue.

- [x] I checked to make sure that this feature has not been requested already.

## 1. The entire URL of the file you are using

https://github.com/tensorflow/models/tree/master/official/...

## 2. Describe the feature you request

I am actually using the EfficientDet Model on AWS SageMaker but what I want to have a NMS layer on the inference because when I train my model the NMS is not applied and also I want is to have the loss and the accuracy not on tensorboard but on text file or on an other thing 

## 4. Are you willing to contribute it? (Yes or No)

No

**Comment by yeqingli** [comment]
Please elaborate the question. What feature do you need? It seems that the EfficentDet is mostly from [ODAPI](https://github.com/tensorflow/models/tree/238922e98dd0e8254b5c0921b241a1f5a151782f/research/object_detection) instead of the official model garden.

Also you might want to check the NMS implementation in TF, such as [tf.image.non_max_suppression](https://www.tensorflow.org/api_docs/python/tf/image/non_max_suppression) or [tf.image.combined_non_max_suppression](https://www.tensorflow.org/api_docs/python/tf/image/combined_non_max_suppression)
